### DISCUSSION TITLE
Новые разновидности заданий

### DISCUSSION THREAD

**Discussion by SnakeLuck** [octothorp-team/public-dobrograd#717] [Предложения]
Я играя на Доброграде столкнулся с проблемой заработка денег, и я решил выполнять задания в F4, но здесь как раз и заключается вся проблема. Всего разновидностей заданий только 4. И по этому я предлагаю добавить разновидности заданий по таким параметрам:
1) Профессия персонажа
2) Количество кармы
3) Время проведенное на проекте
И сейчас я вам объясню как я это вижу. Возьмем к примеру самого обычного новичка который только узнал о существовании проекта и решил поиграть на Доброграде. Зашел на сервер побегал попрыгал и захотел купить оружие/машину, и он потихоньку выполняет обычные задания (продукты, еда, грузы, драгоценности) а затем смотря как поменяется его карма если пойдет в плюс то на +50 у него появятся такие задания:
 Экспресс доставка(Игроку нужно доставить посылку за отведенное время (8/7/6/5 минут));
 Рыбалка (Игроку нужно поймать определенную рыбу(1-4шт.) за отведенное время (от 5мин));
 Обстройка (Игроку  нужно обстроить квартиру с определенным лимитом пропов, за отведенное время ( от 10мин))(после окончания задания пропы или удаляются или же остаются в качестве декора квартиры/дома)); 
Почтальон (Игроку нужно доставить газеты в ЖИЛЫЕ квартиры за отведенное время (от 10мин))
 Далее пойдут задания от +100 кармы:
 Эксклюзивная доставка (Игроку нужно доставить посылку за отведенное время (5/4/3мин); 
 Поставка рыбы ( игроку нужно поймать определенную рыбу(5-10шт) за определенное время ( от 15мин);
 Декоратор ( Игроку нужно обстроить улицу за отведенное время (от 5мин);
Задания для полиции (Не зависят от кармы)
 Патрулирование города (Игроку нужно проехать по определенному пути от ПД к примеру: требуется проехать Рузвельт-Бравери-Бронкстон-Джефферсон; за отведенное время (от 10мин))
 Патрулирование улицы (Игроку нужно пробыть отведенное время на определенной улице ( от 5мин))
 Поиск улик (Игроку нужно прибыть на определенную улицу для поиска оружия/наркотиков они могут как и просто валятся на тротуаре так и быть где-то спрятанными (от 10мин);
 Далее пойдут задания от -50 кармы:
 Курьер (Игроку нужно пронести наркотики/оружие от точки А до точки Б, но если вас поймает полиция то вы отправитесь в тюрьму (проносить придется в своей куртке или же в руках) (от 10мин))
 Держать район (Игроку требуется находится на улице с оружием в инвентаре (от 10мин))
 Угон (Игроку требуется угнать машину и привезти ее в точку Б, после чего у угонщика появляется блокировка на такой тип заданий (от 5мин) (от 10мин))
 Взломщик (Игроку требуется открыть отмычкой определенную дверь (от 10мин))
  Улики (Игроку требуется спрятать улики на определенной улице (от 5мин))
P.S. На данный момент у меня больше нет идей для заданий в f4 но в будущем возможно появятся еще, надеюсь вы сжалитесь над игроками и добавить такой способ заработка ткк деньги довольно сложно получить, как сложно? Просто ждать пока тебе капнет на карман зарплата и всё.
P.S. Да некоторые, а может и большинство заданий глупые и неуместные, но это только первые идеи и задумки, что-то нужно доработать или изменить а может даже и удалить, я не отрицаю. И главное в том чтобы что-то изменить и подкорректировать так чтобы потом это было применено на проекте, это обычная критика и советы, помоги мне и другим авторам в создании чего вместо того чтобы просто обосрать идею, только из-за того что тебе что-то не понравилось. На этом всё,  спасибо за ваше внимание!

**Comment by VaR1aN** [comment]
Абсолютно не нужные задания...

**Reply by SnakeLuck** [reply]
А таскать онли грузы это крутое задание? Или же ты с кем-то закоопился и носишь то продукты, то еду? Конечно нет. И тем более нужны задания для полиции, ткк на профе тупо нечего делать, так и в рыбалке нет смысла, а моя задумка может помочь с преданием смысла рыбалки + смысл быть полицейским
